hubblestack/extmods/returners/splunk_fdg_return.py

```diff
@@ -183,7 +183,7 @@ def returner(ret):
                         fields = {}
                         for item in index_extracted_fields:
                             if item in payload['event'] and not isinstance(payload['event'][item], (list, dict, tuple)):
-                                fields[item] = str(payload['event'][item])
+                                fields["meta_%s" % item] = str(payload['event'][item])
```

Comment:
Why isn't this `fields[item] = payload['event'].pop(item)` again? It had something to do with archiving or something. Seems silly to duplicate the data (with a different name), but I vaguely recall there was a reason.

Reply:
We want it in both so that you can still search for the data in standard event searches. If it's only in meta, it doesn't appear in the event when you're looking through the data.